cmd/minishift/cmd/root.go

```diff
@@ -79,12 +79,12 @@ var RootCmd = &cobra.Command{
 	Long:  `Minishift is a command-line tool that provisions and manages single-node OpenShift clusters optimized for development workflows.`,
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
 		var (
-			err          error
-			isFreshStart bool
+			err                    error
+			isAddonInstallRequired bool
 		)
 
-		if !filehelper.Exists(constants.Minipath) || filehelper.IsEmptyDir(constants.Minipath) {
-			isFreshStart = true
+		if !filehelper.Exists(constants.MakeMiniPath("addons")) {
```

Comment:
@hferentschik , are we taking this path now instead of what we decided as part of PR https://github.com/minishift/minishift/pull/990.